google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java

```diff
@@ -116,4 +121,28 @@ public void testPaginationIterating() {
     }
     Assert.assertTrue(presented);
   }
+
+  @Test
+  public void testPaginationAggregatedIterating() throws IOException {
+    AcceleratorTypesClient acceleratorTypesClient = AcceleratorTypesClient.create();
+    AggregatedListAcceleratorTypesRequest aggregatedListAcceleratorTypesRequest =
+        AggregatedListAcceleratorTypesRequest.newBuilder()
+            .setProject(DEFAULT_PROJECT)
+            .setMaxResults(2)
+            .build();
+    AcceleratorTypesClient.AggregatedListPagedResponse response =
+        acceleratorTypesClient.aggregatedList(aggregatedListAcceleratorTypesRequest);
+    boolean presented = false;
+    for (Map.Entry<String, AcceleratorTypesScopedList> entry : response.iterateAll()) {
+      if (entry.getKey().equals("zones/" + DEFAULT_ZONE)) {
+        for (AcceleratorType type : entry.getValue().getAcceleratorTypesList()) {
+          if (type.getName().equals("nvidia-tesla-t4")) {
+            presented = true;
```

Comment:
did you mean "present"?

Reply:
I used "presented" as a short form of "is presented in the list"